experimental/benchmarks/node-dc-eis/kill_node_linux

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
@kunalspathak Once we get Node-DC-EIS in, I can work to get all the scripts in one place or in fact we can consolidate them (at least external application specific scripts).